server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -262,6 +262,15 @@ public Map<InetSocketAddress, ServerPort> activePorts() {
         }
     }
 
+    /**
+     * Returns the {@link ProjectManager} of the server if the server is started.
+     * {@code null} is returned, otherwise.
+     */
+    @Nullable
+    public ProjectManager projectManager() {
```

Comment:
Hopefully we can deprecate/remove this API later when we expose APIs to update control plane resources later on 🙏